utility/src/main/java/org/oppia/util/parser/GlideImageLoader.kt

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import javax.inject.Inject
+
+class GlideImageLoader @Inject constructor() : ImageLoader {
```

Comment:
Please provide a brief KDoc for this class. E.g. something like the following is sufficient:

```kotlin
/** An [ImageLoader] that uses Glide. */
```

Reply:
Done.